scrapy/commands/shell.py

```diff
@@ -84,7 +85,7 @@ def run(self, args: list[str], opts: Namespace) -> None:
         crawler._apply_settings()
         # The Shell class needs a persistent engine in the crawler
         crawler.engine = crawler._create_engine()
-        crawler.engine.start(_start_request_processing=False)
+        deferred_from_coro(crawler.engine.start_async(_start_request_processing=False))
```

Comment:
`deferred_from_coro()` here and in some other places is used as a shortcut for "if using asyncio, create a task for the coro", as just calling the coro wouldn't schedule it; we could add a private helper function instead, or change all these places to keep the reference to the returned deferred so that the code doesn't look weird (and to add exception handling)